indexer/services/comlink/src/controllers/api/v4/orders-controller.ts

```diff
@@ -114,7 +123,28 @@ class OrdersController extends Controller {
       ),
     ]);
 
-    const postgresOrderMap: PostgresOrderMap = _.keyBy(postgresOrders, OrderColumns.id);
+    const redisOrderIds: string[] = _.map(
+      Object.values(redisOrderMap),
+      (redisOrder: RedisOrder) => {
+        return OrderTable.orderIdToUuid(redisOrder.order!.orderId!);
+      },
+    );
+
+    // Fetch additional orders from Postgres that are not in the initial query, because they
+    // might not conform to the status filter above. For example, if the user is querying for
+    // `status: [BEST_EFFORT_OPENED]`, we need to fetch all orders from Postgres, because if the
+    // order in postgres is 'OPENED', then we do not want to return this order to the user as
+    // 'BEST_EFFORT_OPENED'.
```

Comment:
nit: The main reason for this is because postgres is regarded as the source of truth, and redis and postgres may differ in the status of an order. Can you call this out specifically in the comment?